eth-contracts/contracts/service/DelegateManager.sol

```diff
@@ -104,7 +111,9 @@ contract DelegateManager is RegistryContract {
         serviceProviderFactoryKey = _serviceProviderFactoryKey;
         claimsManagerKey = _claimsManagerKey;
         undelegateLockupDuration = 10;
-
+        maxDelegators = 175;
```

Comment:
maybe we move this to be a function parameter instead of defining as literal? same with minDelegationAmount, undelegateLockupDuration. wdyt?

Reply:
not worth the refactor atm, maybe if we have extra time later. having the right chain value is more important imo